src/lib.rs

```diff
@@ -63,7 +97,7 @@
 #![warn(clippy::print_stdout)]
 
 #[doc = include_str!("../README.md")]
-#[cfg(doctest)]
+#[cfg(all(doctest, feature = "log", feature = "tracing"))]
```

Comment:
This is required to in order to pass the CI check as the Readme contains both log and tracing examples
`cargo hack test --feature-powerset --workspace`